04/styles/desktop.css

```diff
@@ -0,0 +1,73 @@
+/* ./styles/desktop.css */
+
+a {
+	text-align: center;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+header h1 {
+    margin-left: 20px;
+}
+
+.menu, .submenu {
+    display: none;
+}
+
+.nav {
+	height: 40px;
+	width: 400px;
+    line-height: 200%;
+    display: flex;
+}
+
+.nav__item {
+	width: 100px;
+    height: 40px;
+    border: 0px;
+	position: relative;
+}
+
+.nav__item:hover {
+    border-top: 1px solid var(--color-gamma);
```

Comment:
Raczej bym uważał z tego typu rozwiązaniem.
Nie wiem czy taki był u Ciebie zamysł, ale gdy pojawia się border to tekst przesuwa się o 1px.
Czasami jest to bardzo problematyczne. Lepiej domyślnie dać obramowanie z przeźroczystym kolorem.
Potem na `:hover` zmieniasz tylko kolor :)